CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.0.3
+
+* Annotate the type of 'variables' in class All and class None.
```

Comment:
I'd write "Work around a common front-end inference bug." The changelog is intended for downstream users, and as far as they're concerned the only change here is that this now works with the CFE.